### ISSUE TITLE
Crashes in Hill Caves

### ISSUE THREAD

**Issue by ThalionFan** [Pyrdacor/Ambermoon.net#197]
I've been getting a lot of crashes in the Hill Caves and the Save.99 folder doesn't seem to appear for me. The crash seems to be completely arbitrary - I've had it happen in combat, while using items and while moving around the map. The game freezes for a couple of seconds, then closes. The strange thing is that I went through a large part of the Hill Caves last night and had no issues at all, it's only when I went back to the game this afternoon that it started happening. I'm using the English 1.08 version and my OS is 64-bit Windows 10.

**Comment by Pyrdacor** [comment]
Sound like a hardware issue. Maybe your graphic card crashes the game? Do you have the most up-to-date drivers installed?

Is there no error message?

**Comment by ThalionFan** [comment]
No error message - the game just freezes then closes. I've actually just gone back to it and played through the rest of the Hill Caves saving frequently and it's been fine so far. I've been playing on two seperate computers - a 6 month-old laptop and a 6 year-old desktop both running Windows 10 - the desktop is the one where I started getting crashes so it probably is hardware-related. I only moved it to the desktop so I could sit at a proper desk while playing since it's been long sessions so far!
